### PR TITLE
feat :rating and modal

### PR DESCRIPTION
**Pull request by Aazam10**
Ratings and modal components added.
CSB for rating: https://77xvh9.csb.app/components/ratings/ratings.html
CSB for modal: https://77xvh9.csb.app/components/Modal/modal.html

### CODE REVIEW

**Review by Ayushi20-19 — APPROVED** [review]
Clean and Commented code good work

File: components/Modal/modal.css [hunk 1]
```diff
@@ -0,0 +1,44 @@
+:root {
+  --border-color: rgba(0, 0, 0, 0.2);
+  --border-shadow: rgba(0 0 0 / 50%);
+}
```
Comment (kart-c) [inline components/Modal/modal.css:4]: you can make a common stylesheet and add this in it

**Review by kart-c — APPROVED** [review]
The code is well indented, the rating component looks really good

File: components/Modal/modal.css (hunk 1 above)
Comment (Aazam10) [reply components/Modal/modal.css:4]: Ok will do that

**Pull request merged**